engine/hid/src/test/test_app_hid.cpp

```diff
@@ -191,6 +191,7 @@ static void* EngineCreate(int argc, char** argv)
 
 static const char* KeyToStr(dmHID::Key key)
 {
+#if !defined(DM_PLATFORM_VENDOR)
```

Comment:
It was commented out on the private platform.
Better to use defines to control code.